squad/core/migrations/0148_build_patch_url.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.16 on 2020-12-11 10:33
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0147_add_build_and_environment_to_test'),
```

Comment:
Migration dependency needs to be updated. I've added a new one in my last merged PR.

Reply:
done